src/VisualStudio/VisualBasic/Impl/Options/AutomationObject/AutomationObject.vb

```diff
@@ -0,0 +1,92 @@
+﻿' Licensed to the .NET Foundation under one or more agreements.
+' The .NET Foundation licenses this file to you under the MIT license.
+' See the LICENSE file in the project root for more information.
+
+Imports System.Runtime.InteropServices
+Imports Microsoft.CodeAnalysis
+Imports Microsoft.CodeAnalysis.CodeStyle
+Imports Microsoft.CodeAnalysis.Options
+
+Namespace Microsoft.VisualStudio.LanguageServices.VisualBasic.Options
+    <ComVisible(True)>
+    Partial Public Class AutomationObject
+        Private ReadOnly _workspace As Workspace
+
+        Friend Sub New(workspace As Workspace)
+            _workspace = workspace
+        End Sub
+
+        Private Function GetBooleanOption(key As PerLanguageOption2(Of Boolean)) As Boolean
```

Comment:
can these helpers not be shared?  have a common AbstractAutomationObject for both langauges?

Reply:
I'll try to share "as much as I can". GetBooleanOption cannot be shared since it returns a boolean for one language and an integer for the other. Unifying the return value is potentially a breaking change.